backend/plugins/workflow_manager/workflow_v2/utils.py

```
@@ -61,25 +61,66 @@ def add_file_destination_filehash(
         """
         return file_hash
 
+    @staticmethod
+    def validate_rule_engine(
+        result: Any | None,
```

Ruff (0.14.8)

Unused static method argument: `result`

(ARG004)